packages/frontend-design-poc/src/components/FilterBar/AddFilterButton/AddFilterButton.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button } from '@digdir/designsystemet-react';
+import { PlusIcon } from '@navikt/aksel-icons';
+import { useTranslation } from 'react-i18next';
+import { Filter, FilterBarField } from '../FilterBar.tsx';
+import { FilterList, FilterListItem } from '../FilterList';
+
+import styles from './addFilterButton.module.css';
+
+type AddFilterButtonProps = {
+  filterOptions: FilterBarField[];
+  filters: Filter[];
+  onListItemClick: (option: FilterBarField) => void;
+  onBtnClick: () => void;
+  isOpen: boolean;
+  disabled?: boolean;
+};
+export const AddFilterButton = ({
+  filterOptions,
+  onListItemClick,
+  disabled,
+  filters,
+  onBtnClick,
+  isOpen,
+}: AddFilterButtonProps) => {
+  const { t } = useTranslation();
+  return (
+    <div>
+      <Button size="small" onClick={onBtnClick} disabled={disabled} variant="secondary" color="first">
+        <PlusIcon /> {t('filter_bar.add_filter')}
+      </Button>
+      {isOpen && (
```

Comment:
Potentially refactor this into a `<ButtonWithDropdown />`-component? To extract the button, isOpen and dropdown functionality.

Reply:
Vi må se hvor gjenbrukbart det kan bli på sikt, og behovene. Ser ingen verdi i å skille ut dette som egen komponent foreløpig.